### ISSUE TITLE
diff mode

### ISSUE THREAD

**Issue by VictorBusch** [zevv/duc#315]
Hello,

to find growing files and directories, a diff mode would be interesting.
It was not planned in 2019. What about 2023/4? ;-)

Best regards

**Comment by zevv** [comment]
Yeah, that would be nice indeed. but I'm sorry, new Duc developments have mostly stalled - it serves the original purpose well and scratches my personal itches, and I'm not really willing to invest much more time into this.

There's a few big things that would be very nice to have but will probably never make it into the current code base, like threaded` indexing, diff modes, incremental indexing.
